src/components/Loading/Loading.styles.js

```diff
@@ -0,0 +1,45 @@
+import styled from '@emotion/styled';
+import { css, keyframes } from '@emotion/core';
+
+export const Wrapper = styled.div`
+  align-items: center;
+  display: flex;
+`;
+
+export const Spinner = styled.div`
+  height: 2.5rem;
+  margin: auto;
+  position: relative;
+  width: 2.5rem;
+`;
+
+const doubleBounceStyles = ({ theme }) => css`
+  animation: ${bounce} 2s infinite ease-in-out;
+  background: ${theme.color.indigo600};
+  border-radius: 50%;
+  height: 100%;
+  left: 0;
+  opacity: 0.6;
+  position: absolute;
+  top: 0;
+  width: 100%;
+`;
+
+export const DoubleBounceOne = styled.div`
+  ${doubleBounceStyles}
+`;
+
+export const DoubleBounceTwo = styled.div`
```

Comment:
```suggestion
export const DoubleBounce = styled.div`
  ${doubleBounceStyles}
`;

export const DoubleBounceWithDelay = styled.div`
```

Reply:
What do you think about those names? I think that is more clear when you read them from the component in this way, so you can easier guess what is the difference between them. Thoughts?